src/Tribe/Views/V2/View.php

```diff
@@ -755,37 +755,25 @@ public function next_url( $canonical = false, array $passthru_vars = [] ) {
 
 		$url = $this->get_url();
 
-		$query_args = [];
-
 		if ( ! empty( $passthru_vars ) ) {
 			// Remove the pass-thru vars, we'll re-apply them to the URL later.
 			$url = remove_query_arg( array_keys( $passthru_vars ), $url );
 		}
 
 		// Make sure the view slug is always set to correctly match rewrites.
-		$query_args['eventDisplay'] = $this->slug;
+		$url = add_query_arg( [ 'eventDisplay' => $this->slug ], $url );
 
-		if ( $this->has_next_event ) {
-			$query_args[ $this->page_key ] = $this->url->get_current_page() + 1;
-		} else {
-			$query_args = [];
-			$url        = '';
-		}
+		$url = $this->has_next_event ?
+			add_query_arg( [ $this->page_key => $this->url->get_current_page() + 1 ], $url )
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: e854023f372890e1fa11b2eb5c99071f</sub>